dist/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
The project looks awesome and meets all technical requirements. Please update your README. It should be professional with instructions to set the project up locally. Please add an active live demo link or remove this option.